### PR TITLE
Donation page

### PR DESCRIPTION
**Pull request by stefanb**
Implementation using Stripe and VueStripe library - https://github.com/vue-stripe/vue-stripe
https://yarnpkg.com/package/@vue-stripe/vue-stripe

### CODE REVIEW

File: src/pages/DonationPage.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<script>
+import { StripeCheckout } from '@vue-stripe/vue-stripe';
+
+export default {
+  name: 'DonationPage',
+  components: {
+      StripeCheckout
+  },
+  data () {
+    this.publishableKey = 'pk_test_51I6cVwEUtJJGJFIUgsLpXxbk4DHr2g8Q2ZCIUlzSWbVBtWbcEZJdJuRhyjCxO0xFZdBecREiHx2zirBDPFN1qoeQ00A5Wd3bak' // process.env.STRIPE_PUBLISHABLE_KEY;
```
Comment (stefanb) [inline src/pages/DonationPage.vue:30]: Tole je potrebno premakniti v `.env` konfiguracije in nastaviti produkcijski key ko bomo srečni z implementacijo

File: src/pages/DonationPage.vue [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        {
+          amount: 100,
+          lineItems:[
+            {
+              price: 'price_1I6dUFEUtJJGJFIUcWNRbSDC', // The id of the recurring price you created in your Stripe dashboard
+              quantity: 1,
+            },
+          ]
+        },
+      ],
```
Comment (stefanb) [inline src/pages/DonationPage.vue:107]: Tu imamo definirane subscription pakete

**Comment by overlordtm** [comment]
Why is 100EUR written with word while other amounts are in numbers? Also, `STO` is not translate don english page.

File: .env [hunk 3]
```diff
@@ -4,3 +4,5 @@ VUE_APP_DESC='COVID-19 Sledilnik Slovenija - zbrani, preverjeni, ažurirani, ana
 VUE_APP_URL='https://covid-19.sledilnik.org'
 VUE_APP_DEFAULT_LANGUAGE='sl'
 VUE_APP_LOCALE_CONTEXT='SVN'
+VUE_APP_STRIPE_PUBLISHABLE_KEY='pk_test_51I6cVwEUtJJGJFIUgsLpXxbk4DHr2g8Q2ZCIUlzSWbVBtWbcEZJdJuRhyjCxO0xFZdBecREiHx2zirBDPFN1qoeQ00A5Wd3bak'
+VUE_APP_STRIPE_SUBSCRIPTIONS='5:price_1I6dUEEUtJJGJFIUM4slrZ9g;20:price_1I6dUEEUtJJGJFIURHozzi1V;50:price_1I6dUEEUtJJGJFIUnmagHiRR;STO:price_1I6dUFEUtJJGJFIUcWNRbSDC'
```
Comment (stefanb) [inline .env:8]: @overlordtm "STO" comes from config and is different in env.production

Comment (stefanb) [reply .env:8]: We could also change it:
```suggestion
VUE_APP_STRIPE_SUBSCRIPTIONS='5:price_1I6dUEEUtJJGJFIUM4slrZ9g;20:price_1I6dUEEUtJJGJFIURHozzi1V;50:price_1I6dUEEUtJJGJFIUnmagHiRR;100:price_1I6dUFEUtJJGJFIUcWNRbSDC'
```
but i wanted some visual indication that it differs from production config

Comment (stefanb) [reply .env]: synced with prod settings in https://github.com/sledilnik/website/pull/793/commits/eefbab1e906a72bd193afbbf564b0bad935b1c42

**Pull request merged**